src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -0,0 +1,1108 @@
+namespace System.Security {
```

Comment:
Please use our genapi tool to generate this ref file so that it is initially consistent.